### PR TITLE
nerfs styptic/silversulf/synthflesh to have overdoses

### PR DESCRIPTION
**Pull request by Poojawa**
## About The Pull Request

This PR seeks to end the complete dominance that Silver Sulf and Styptic powder hold over all chemical based healing. Both chems have recieved metabolism and overdose drawbacks, but It's still very generous to both.

Oxandrolone/Salicyclic acid was given a health range buff of 50 to 25 to  make it more feasible and common in medical treatments. They still carry low OD thresholds.

Synthflesh was also given a metabolism rate. as well as application drawbacks. The 1:1 descriptor of healing is now actually truthful, rather than the 1.25:1 it was

## Why It's Good For The Game

Unless something changes, the vast majority of chems available for mixing are completely moot, due to the as of now lack of any drawbacks or issues. Just stock up on hundreds of units of the usually three to receive massive healing overtime. Literally every other chemical has some form of caution you must keep in mind, why not these three?

Now, overdoses will not only cause side effects, but they also purge the chemical three times faster.

## Changelog
:cl: Poojawa
balance: Chemistry is encouraged to be more varied in their healing, as well as careful in its application
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -283,10 +294,12 @@
 /datum/reagent/medicine/styptic_powder
 	name = "Styptic Powder"
 	id = "styptic_powder"
-	description = "If used in touch-based applications, immediately restores bruising as well as restoring more over time. If ingested through other means, deals minor toxin damage."
+	description = "If used in touch-based applications, immediately restores bruising as well as restoring more over time. If poisonous if applied orally or injection. If overdosed, deals brute and minor liver damage."
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: accidental extra if here on **"If poisonous..."** doesn't really matter much but yuh

Comment (Poojawa) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: good catch.

**Comment by Weblure** [comment]
I honestly had no idea that you couldn't OD with these two chemicals. I think this is a good balance change that should've been present from the start. The only risk-free healing options should be the ones in the standard medkits (balanced by having to apply to individual body parts) and ones that aren't easy to obtain/create round-start, such as healing jelly and synthflesh.

Edit: I just realized you added an OD threshold to synthflesh. I think a threshold of 40 is a bit too low. Synthflesh should be able to heal someone that's in heavy crit to at least a decent amount of positive health without the risk of overdosing them. Is it possible for the OD threshold to last for a longer period of time without slowing the metabolism rate of reagent itself? That'd be a good way to prevent people from abusing synthflesh to infinitely heal themselves in combat without nerfing its usefulness for medical personnel.

Also, does this OD threshold apply to corpses? If so, that'd make it impossible to heal people for revival surgery without ODing the hell out of them.

**Comment by Poojawa** [comment]
all the OD does is make synthflesh metabolize super quick, getting out of the system quicker

**Comment by Poojawa** [comment]
*scream

**Comment by Thalpy** [comment]
I've sort of fiddled with these already in an upcoming pr that relies on 2 in review prs to be merged, and a 3rd pr from those 2 which I'll put up when those PRs are merged.

But anyways, when those 3 PRs do get merged I'll try to edit it so it keeps what you've done here, but also uses the purity effects, so hopefully that's okay. I just thought I'd let you know.

I thought atomising was supposed to make things less confusing.

**Comment by Poojawa** [comment]
Atomizing at this point is code for 'I don't want to deal with your big ass PR right now' from what I can tell. lol

**Review by silicons — APPROVED** [review]

**Pull request merged**